cassandra-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraExecutor.java

```diff
@@ -343,6 +379,9 @@ private void safeShutdown() {
                 // ignore this
             }
             process = null;
+
+            // send health check that process is no longer alive
+            healthCheck(driver);
```

Comment:
What is the motivation for sending a healthcheck message when we're shutting down the server.  Isn't "shut down" and "unhealthy" two different things in the scenario that the scheduler asked that the process be shut down?